.github/actions/publish/action.yml

```diff
@@ -4,6 +4,9 @@ inputs:
   token:
     description: 'Token to use for publishing.'
     required: true
+  homebrew-gh-secret:
+    description: 'Token to use for publishing to homebrew-tap.'
```

Comment:
Technically an SSH private key and not a token, we should probably clarify that here